robocup_knowledge/src/robocup_knowledge/environments/rgo2019/challenge_serving_drinks.py

```diff
@@ -0,0 +1,16 @@
+# SERVING DRINKS KNOWLEDGE FILE RGO2019
+
+# Bar where the drinks are located
+bar_id = "bar_table"
+
+# Room where the party takes place, i.e., where the robot should look for people
+room_id = "living_room"
+
+# Default guest
+operator_name = "default_guest"
+
+# Starting location
+starting_point = "serving_drinks_initial"
+
+# Number of drinks to be served
+NR_DRINKS = 3
```

Comment:
Don't use capital letters, it is used for global constants.

Reply:
Will be reflected in the new PR (styling)